lib/tasks/mailchimp.rake

```diff
@@ -0,0 +1,71 @@
+require "digest"
+require "MailchimpMarketing"
+
+namespace :mailchimp do
+  desc "Synchronise Mailchimp audiences with the users in the database"
+  task synchronize_audiences: :environment do
+    mailchimp_api_key = ENV["MAILCHIMP_API_KEY"]
+    mailchimp_server_prefix = ENV["MAILCHIMP_SERVER_PREFIX"]
+    mailchimp_lists = ENV["MAILCHIMP_LISTS"]
```

Comment:
Could we get these from the normal [settings](https://github.com/alphagov/forms-admin/blob/main/config/settings.yml) [mechanism](https://github.com/rubyconfig/config) rather than directly from the environment?

Reply:
No worries :) It's a thing we've added to our apps, it's not part of Rails out-of-the-box.